### PR TITLE
Add missing age and sex to participants.tsv

### LINKED ISSUE #10: missing age and sex for some subjects in participants.tsv
Not all subjects in [participants.tsv](https://github.com/spine-generic/data-multi-subject/blob/master/participants.tsv) file have listed age. Age could be used as a covariate in statistical analysis to rule out potential effect of age on results etc.
Most of subjects with missing age also miss sex.

Sites with missing age:

**institution_id | institution**
- [x] amu | AMU - CEMEREM
- [x] beijing750 | Beijing Tiantan Hospital - sex and age added, date of scan still missing
- [x] beijingPrisma | Beijing Tiantan Hospital - sex and age added, date of scan still missing
- [x] beijingVerio | Beijing Tiantan Hospital - sex and age added, date of scan still missing
- [x] brnoUhb | The University Hospital Brno, Department of Radiology and Nuclear Medicine, Czech Republic
- [x] dresden | Institut_fur_Neuroradiologie, Dresden, Germany
- [x] fslAchieva | Santa Lucia Foundation IRCCS, Rome, Italy
- [x] fslPrisma | Santa Lucia Foundation IRCCS, Rome, Italy - sex and age added, date of scan still missing
- [x] vallHebron | Vall d'Hebron Research Center

### LINKED ISSUE #27: Fix participants.tsv after fsl datasets modifications

### PR DESCRIPTION
**Pull request by valosekj**
This PR is intended for adding of missing age, sex (and date of scan) information to participants.tsv file.

Fixes: spine-generic#10, Fixes #27
(Replaces incorrectly opened PR https://github.com/valosekj/data-multi-subject/pull/1)

### CODE REVIEW

File: participants.tsv [hunk 1]
```diff
@@ -240,13 +240,13 @@ sub-unf04	M	31	2018-12-07	unf	"Neuroimaging Functional Unit (UNF), CRIUGM, Polyt
...
-sub-vallHebron06	M	-	2019-01-25	vallHebron	Vall d'Hebron Research Center	Siemens	TrioTim	NeckMatrix	syngo_MR_B17	Deborah Pareto Onghena
-sub-vallHebron07	M	-	2019-03-04	vallHebron	Vall d'Hebron Research Center	Siemens	TrioTim	NeckMatrix	syngo_MR_B17	Deborah Pareto Onghena
+sub-vallHebron01	M	24	2019-01-16	vallHebron	Vall d'Hebron Research Center	Siemens	TrioTim	NeckMatrix	syngo_MR_B17	Deborah Pareto Onghena
+sub-vallHebron02	F	26	2019-01-16	vallHebron	Vall d'Hebron Research Center	Siemens	TrioTim	NeckMatrix	syngo_MR_B17	Deborah Pareto Onghena
+sub-vallHebron03	F	21	2019-01-23	vallHebron	Vall d'Hebron Research Center	Siemens	TrioTim	NeckMatrix	syngo_MR_B17	Deborah Pareto Onghena
+sub-vallHebron04	M	33	2019-01-25	vallHebron	Vall d'Hebron Research Center	Siemens	TrioTim	NeckMatrix	syngo_MR_B17	Deborah Pareto Onghena
+sub-vallHebron05	F	29	2019-01-21	vallHebron	Vall d'Hebron Research Center	Siemens	TrioTim	NeckMatrix	syngo_MR_B17	Deborah Pareto Onghena
+sub-vallHebron06	M	25	2019-01-25	vallHebron	Vall d'Hebron Research Center	Siemens	TrioTim	NeckMatrix	syngo_MR_B17	Deborah Pareto Onghena
```
Comment (valosekj) [inline participants.tsv]: sub-vallHebron06 is here, in participants.tsv file, stated as a male (M), however, in xls file in https://github.com/spine-generic/data-multi-subject/issues/10#issuecomment-683888404 is stated as a female (F)

Comment (jcohenadad) [reply participants.tsv]: this is highly problematic. it suggests that every entry could be wrong. we need to follow up with the researchers to get to the source (dicoms)

**Comment by jcohenadad** [comment]
@valosekj can we move forward with this PR? it's been in "draft" stage for a while, causing confusion (e.g. recent duplicate: https://github.com/spine-generic/data-multi-subject/pull/47)

**Comment by jcohenadad** [comment]
~@valosekj @alexfoias desden site is not answering-- let's merge and we will add dresden later~

nevermind-- info added in the issue

**Comment by valosekj** [comment]
> @valosekj can we move forward with this PR? it's been in "draft" stage for a while, causing confusion (e.g. recent duplicate: #47)

Sorry. I was waiting for dresden site. I have just added info for this side based on your last https://github.com/spine-generic/data-multi-subject/issues/10#issuecomment-690434010 and merged master into this branch. After checks, this PR should be ready to merge.

**Review by jcohenadad — APPROVED** [review]
thank you!

File: participants.tsv (hunk 1 above)
Comment (jcohenadad) [reply participants.tsv]: @alexfoias can you please follow up on that

**Comment by jcohenadad** [comment]
Duplicate of https://github.com/spine-generic/data-multi-subject/pull/50

**Pull request closed without merging**